src/scenario.py

```diff
@@ -126,6 +128,11 @@ def run(self, strategy_name, options):
                 curLoad += gc_load
 
                 gcPowerSchedule[gcID].append(gc.target)
+                if gc.window:
+                    window = 100
+                else:
+                    window = 0
```

Comment:
Instead of 0 or 100, make it 0 or 1 and scale it with max(total_power) or max(schedule) when plotting